flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -37,6 +37,14 @@ echo "Flink distribution directory: $FLINK_DIR"
 
 EXIT_CODE=0
 
```

Comment:
I would recommend to move it to  the nightly tests. Queryable state is not a core component and the normal builds are already timing out.